src/main/java/org/folio/invoices/utils/ErrorCodes.java

```diff
@@ -60,6 +60,7 @@ public enum ErrorCodes {
   CANCEL_TRANSACTIONS_ERROR("cancelTransactionsError", "Failed to cancel transactions"),
   ERROR_UNRELEASING_ENCUMBRANCES("errorUnreleasingEncumbrances", "Error unreleasing encumbrances after cancelling the invoice"),
   MULTIPLE_FISCAL_YEARS("multipleFiscalYears", "Multiple fiscal years are used with the funds %s and %s."),
+  MULTIPLE_FISCAL_YEARS_ERROR("multipleFiscalYearsError", "Multiple fiscal years are used with codes %s and %s."),
```

Comment:
need to be renamed, maybe MULTIPLE_ADJUSTMENTS_FISCAL_YEARS("multipleAdjustmentsFiscalYears")

Reply:
renamed